testground/benchmark/benchmark/stateless.py

```diff
@@ -396,9 +396,11 @@ def wait_for_peers(home: Path):
     cfg = tomlkit.parse((home / "config" / "config.toml").read_text())
     peers = cfg["p2p"]["persistent_peers"]
     for peer in peers.split(","):
-        host = peer.split("@", 1)[1].split(":", 1)[0]
-        print("wait for peer to be ready:", host)
-        wait_for_port(ECHO_SERVER_PORT, host=host, timeout=2400)
+        parts = peer.split("@", 1)
```

Comment:
`@` is to separate `node-id`, it should always be there?

Reply:
but persistent_peers is empty in this case

Reply:
```
>>> ''.split(',')
['']
```

you mean like this, the `peer` is an empty string?